SU-SWS/stanford_paragraph_types
# EARTH-296 Callout Filmstrip Paragraph Type

# READY FOR REVIEW

# Summary
- stanford_paragraph_callout_filmstrip paragraph type.

# Needed By (Date)
- End of Sprint

# Steps to Test

1. Checkout this branch and the same branch in [SE3_BLT](https://github.com/stanford-earth/SE3_BLT/pull/34) and [stanford_components](https://github.com/SU-SWS/stanford_components/pull/69)
2. config import
3. create complex page with callout filmstrip paragraph type
4. verify it looks correct.

# Associated Issues
- https://github.com/stanford-earth/SE3_BLT/pull/34
- https://github.com/SU-SWS/stanford_components/pull/69

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
